src/main/java/frc/robot/utilities/pid/PIDFTalonSRX.java

```diff
@@ -39,6 +39,16 @@ public PIDFCoefs getCoefs() {
         return pidfCoefs;
     }
 
+    @Override
+    public double getError() {
+        return getClosedLoopError();
+    }
+
+    @Override
+    public double getDeltaError() {
+        return getErrorDerivative();
+    }
+
```

Comment:
remove and put this in PIDFMotor instead

Reply:
that is an interface, it need to be implented into a class

Reply:
PIDFTalonSRX is not an interface. If you meant PIDFMotor, it can be a default method.